lib/model.js

```diff
@@ -3143,7 +3143,7 @@ class Model {
       }
       return values;
     }
-    return this.dataValues;
+    return Utils._.clone(this.dataValues);
```

Comment:
Right, I meant `toJSON` should return cloned data. Changing JSON representation should not change internal model stores.